static/src/stylesheets/module/content-garnett/_types.scss

```diff
@@ -50,4 +52,194 @@
     }
+
+    .content__article-body {
+        @include mq(desktop) {
+            border-left: solid 1px $neutral-5;
+            padding-left: $gs-gutter/2;
+            margin-left: 0;
+        }
+        @include mq(leftCol) {
+            margin-left: -$gs-gutter/2;
+        }
+    }
+
+    .badge,
+    .content__meta-container {
+        @include mq(leftCol) {
+            width: gs-span(2) -$gs-gutter/2;
+        }
+        @include mq(wide) {
+            width: gs-span(3) -$gs-gutter/2;
+        }
+    }
+    // Quote base styles
+    .element-pullquote {
+        background-color: $neutral-garnett-1;
+        border: solid 1px $neutral-5;
+        margin: $gs-gutter/4 0 $gs-baseline*2;
+        padding: $gs-baseline/2 $gs-gutter/2 $gs-baseline;
+        position: relative;
+        clear: none;
+        float: none;
+        width: inherit;
+        @include mq(mobile) {
+            margin-left: -$gs-gutter/2 - 1px;
+        }
+        @include mq(mobileLandscape) {
+            margin-right: $gs-gutter/2;
+            margin-left: -$gs-gutter;
+        }
+        @include mq(leftCol) {
+            margin-left: -$gs-baseline*2;
+        }
+        // Quote tail
+        &:after {
+            @include mq(mobile) {
+                content: '';
+                width: $quote-tail;
+                height: $quote-tail;
+                border-radius: 0 0 $quote-tail;
+                position: absolute;
+                bottom: -$quote-tail - 2px;
+                left: $gs-gutter/2;
+                background-color: $neutral-garnett-1;
+                border: solid 1px $neutral-5;
+                border-top: solid 1px $neutral-garnett-1;
+            }
+            @include mq(mobileLandscape) {
+                left: $gs-gutter;
+            }
+            @include mq(desktop) {
+                left: $gs-gutter - $gs-baseline;
+            }
+            @include mq(leftCol) {
+                left: $gs-baseline;
+            }
+        }
+
+        .pullquote-cite,
+        .pullquote-paragraph {
+            display: inline;
+        }
+
+        .inline-quote,
+        blockquote {
+            display: inline;
+        }
+        // Hack hide for svg
+        .closing {
+            display: none;
+        }
+    }
+    // Quote supporting
+    .element-pullquote.element--supporting {
+        @include mq(mobileMedium) {
+            clear: left;
+            float: left;
+            margin-right: $gs-gutter/2;
+            width: gs-span(3);
+        }
+        @include mq(leftCol) {
+            margin-left: - gs-span(3)/2 - $gs-gutter/2;
+        }
+
+        &:after {
+            @include mq(leftCol) {
+                border-radius: 0 0 0 $quote-tail;
+                left: gs-span(3)/2 - 28px;
+            }
+        }
+
+        .pullquote-cite,
+        .pullquote-paragraph {
+            @include fs-headlineGarnett(1);
+        }
+    }
+    // Quote halfWidth
+    .element-pullquote.element--halfWidth {
```

Comment:
think we tend to kebab-case classes, e.g. `element-pullquote.element--half-width`

Reply:
`element--halfWidth` is a slightly ambiguous name, is it used elsewhere?

Reply:
see below